modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -59,10 +59,10 @@ public class {{classname}}{{#parent}} extends {{{parent}}}{{/parent}}{{^parent}}
   {{#isContainer}}
   {{#useBeanValidation}}@Valid{{/useBeanValidation}}
   {{#openApiNullable}}
-  private {{>nullableDataType}} {{name}} = {{#isNullable}}JsonNullable.undefined(){{/isNullable}}{{^isNullable}}{{#required}}{{{defaultValue}}}{{/required}}{{^required}}null{{/required}}{{/isNullable}};
+  private {{>nullableDataType}} {{name}}{{#isNullable}} = JsonNullable.undefined(){{/isNullable}}{{^isNullable}}{{#defaultValue}} = {{{.}}}{{/defaultValue}}{{/isNullable}};
   {{/openApiNullable}}
   {{^openApiNullable}}
-  private {{>nullableDataType}} {{name}} = {{#required}}{{{defaultValue}}}{{/required}}{{^required}}null{{/required}};
```

Comment:
If the value is removed when `required` is true - does Jackson or something else validate it is set at some point? 

Otherwise, will responses to clients will fail because a required field is missing?

I do realize that required means that a default value shouldn't be used. Just trying to think of how the error scenario is detected.

Reply:
looks like `@NotNull` should be used instead per https://stackoverflow.com/a/45877203/677735

(I think it's out of scope of this PR which aims to fix the default value)